docs/src/surfaces/card/card.jsx

```diff
@@ -42,7 +41,7 @@ export default class NestedCardSample extends React.Component {
     render() {
         return (
             <Card nest={true}>
```

Comment:
Should just be `nest` instead of `nest={true}`.  Non-Blocking, but I'd have thought ESLINT would be buggin' ;-)

Reply:
Fixed.

Reply:
Looks good.  Resolving.